porfile.html

```diff
@@ -56,6 +56,196 @@ <h1 class="title">
+                </li>
+                <li>
+                    <img src="Assets/angellist.svg" alt="Angellist icon">
+                </li>
+            </ul>
+            
+            <button class="aboutme-button">Get my resume</button>
+        </div>
+        <div id="skills">
+            <ul>
+                <li class="skill-title">
+                    <div class="skill-list">
+                        Languages
+                        <img src="Assets/down-arrow.svg" alt="Down arrow">
+                    </div>
+                    <ul>
+                        <li class="skill-detail">
+                            <img src="Assets/javascript.svg" alt="Javascript image">
+                            <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a>
+                        </li>
+                        <li class="skill-detail">
+                            <img src="Assets/html.svg" alt="HTML image">
+                            <a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a>
+                        </li>
+                        <li class="skill-detail">
+                            <img src="Assets/css.svg" alt="CSS image">
+                            <a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a>
+                        </li>
+                    </ul>
+                </li>
+                <li class="skill-title" style="margin-top: 24px;">
+                    <div class="skill-list">
+                        Frameworks
+                        <img src="Assets/right-arrow.svg" alt="Down arrow">
+                    </div>
+                </li>
+                <hr>
+                <li class="skill-title">
+                    <div class="skill-list">
+                        Skills
+                        <img src="Assets/right-arrow.svg" alt="Down arrow">
+                    </div>
+                </li>
+                <hr style="margin-bottom: 90px;">
+            </ul>
+        </div>
+    </section>
+    <section id="contact-us">
+        <h1 class="contact-form-title">
+            Contact me
+        </h1>
+        <p class="contact-form-description">
+            If you have an application you are interested in developing, a feature that you need built or a project that
+            needs
+            coding. I'd love to help with it
+        </p>
+        <form class="contact-form">
+            <input type="text" class="input" value="Samuel Ogoigbe" name="name" placeholder="Enter Name">
+            <input type="email" class="input" value="ogoigbe13@gmail.com" name="email" placeholder="Enter Email">
+            <textarea name="description" class="textarea" cols="30" rows="7"
+                placeholder="Write your message here"></textarea>
+            <button type="button" class="contact-form-button">Get in touch</button>
+        </form>
+    </section>
+    <section class="works"></section>
```

Comment:
- [ ] [ OPTIONAL] Kindly remove the contact form you will work on it after with more documentation :+1: